nox/main.py

```diff
@@ -155,17 +150,25 @@ def filter_sessions_by_keywords(keywords, available_sessions):
 
 def print_summary(results):
     logger.warning('Ran multiple sessions:')
-    for session, result in results:
-        log = logger.success if result else logger.error
+    for session, status in results:
+        if status == SessionStatus.SUCCESS:
+            log = logger.success
+            status = 'passed'
+        elif status == SessionStatus.SKIP:
+            log = logger.warning
+            status = 'skipped'
+        else:
+            log = logger.error
+            status = 'failed'
```

Comment:
I would prefer instead `elif status == SessionStatus.FAIL` and then an `else` branch that did the equivalent of a panic in Golang